lib/vmdb/loggers.rb

```diff
@@ -130,7 +130,10 @@ def self.create_loggers
     private_class_method :create_loggers
 
     def self.create_multicast_logger(log_file_path, logger_class = VMDBLogger)
-      MulticastLogger.new(logger_class.new(log_file_path)).tap do |l|
+      logger_instance = logger_class.new(log_file_path).tap do |logger|
+        logger.level = MulticastLogger::DEBUG
```

Comment:
Should this be `Logger::DEBUG`?

Reply:
Yeah, no reason why not.  I can change in the morning.